controlpanel/frontend/static/components/navbar/macro.html

```diff
@@ -1,19 +1,37 @@
-{% macro navbar(params) %}
-<nav class="hmcts-sub-navigation {{ params.classes or "" }}"
-     {%- for key, value in params.attributes %} {{key}}="{{value}}"
-     {%- endfor %}>
-  <ul class="hmcts-sub-navigation__list">
-  {% for item in params["items"] %}
-    {% if item.href and not item.hide %}
-    <li class="hmcts-sub-navigation__item">
-      <a class="hmcts-sub-navigation__link"
-         {%- if item.active %} aria-current="page"
-         {%- endif %} href="{{ item.href }}"
-         {%- for key, value in item.attributes %} {{key}}="{{value}}"
-         {%- endfor %}>{{ item.html | safe if item.html else item.text }}</a>
-    </li>
-    {% endif %}
-  {% endfor %}
-  </ul>
-</nav>
+{% macro mojPrimaryNavigation(params) %}
+<div class="moj-primary-navigation {{- ' ' + params.containerClasses if params.containerClasses else ''}}">
+
```

Comment:
Does this need so many blank lines?

Reply:
This is copied from [MOJ Frontend](https://github.com/ministryofjustice/moj-frontend/blob/master/src/moj/components/primary-navigation/template.njk) with minor changes to make it work with Jinja. Probably the fewer differences, the better.

Reply:
:+1: